app/res/layout/screen_login.xml

```diff
@@ -86,21 +86,19 @@
                                 android:textSize="@dimen/text_medium"
                                 android:visibility="gone"/>
 
-                            <EditText
+                            <AutoCompleteTextView

```

Comment:
did we determine that there was no spinner that would do this based on concrete values? Autocomplete doesn't necessarily seem significantly better to me, since you still have to start typing, and it covers up the view below